staker/staker.go

```diff
@@ -156,6 +157,7 @@ var DefaultL1ValidatorConfig = L1ValidatorConfig{
 	ExtraGas:                  50000,
 	Dangerous:                 DefaultDangerousConfig,
 	ParentChainWallet:         DefaultValidatorL1WalletConfig,
+	LogQueryRange:             999,
```

Comment:
We want the default value to be zero, which would indicate disabling of this feature.

Reply:
Renamed it!

Reply:
Done!